README.md

```diff
@@ -0,0 +1,144 @@
+# Liana
+
+*The missing safety net for your bitcoins*.
```

Comment:
*a hold to save you if you fall*

Reply:
Oh sorry i missed it, didn't mean to merge without answering it. :sweat_smile: 

I prefer my catchprase though. We can bring it up with the others.